pkgs/tools/misc/yubikey-manager/default.nix

```diff
@@ -55,14 +55,14 @@ python3Packages.buildPythonPackage rec {
     makefun
   ];
 
-  meta = with lib; {
+  meta = {
     homepage = "https://developers.yubico.com/yubikey-manager";
     changelog = "https://github.com/Yubico/yubikey-manager/releases/tag/${version}";
     description = "Command line tool for configuring any YubiKey over all USB transports";
 
-    license = licenses.bsd2;
-    platforms = platforms.unix;
-    maintainers = with maintainers; [ benley lassulus pinpox nickcao ];
+    license = lib.licenses.bsd2;
+    platforms = lib.platforms.unix;
+    maintainers = with lib.maintainers; [ benley lassulus pinpox nickcao ];
```

Comment:
Is there a reason to move `lib` here?

Reply:
Yes, there is, @pinpox 🙂 Among other things #299255 mentions:

> 3. Discouraging nested and toplevel `with`, especially `meta = with lib; {` which was written in nearly every package

Reply:
Okay! Seems I missed that one, thanks for clarifying :)

Reply:
Sure thing, @pinpox, thanks for asking; always helpful to re-validate best-practices that were picked up somewhere in the fast paced development of nixpkgs 🙂

Reply:
> Yes, there is, @pinpox 🙂 Among other things #299255 mentions:
> 
> > 3. Discouraging nested and toplevel `with`, especially `meta = with lib; {` which was written in nearly every package
> 
> 

No, no, no, that’s not a reliable source, it’s just something I came up with from my memory! Some committers still have objections to this change, and it may not be enforced in the end.